src/layouts/Tutorial.tsx

```diff
@@ -222,16 +227,12 @@ export const TutorialLayout = ({
             pt={8}
           />
           {children}
-          {frontmatter.lang !== "en" ? (
-            // TODO: Implement CrowdinContributors
-            <>
-              {/* <CrowdinContributors
-                relativePath={relativePath}
-                editPath={absoluteEditPath}
-                //@ts-ignore
-                langContributors={allCombinedTranslatorsJson.nodes}
-              /> */}
-            </>
+          {frontmatter.lang !== DEFAULT_LOCALE ? (
+            <CrowdinContributors
+              relativePath={relativePath}
+              lastUpdatedDate={intlLastEdit}
+              contributorsByLocale={crowdinContributors}
```

Comment:
Why do we call this `contributorsByLocale`? we are always fetching 1 locale, right? if so, this should be called just `contributors`.

```ts
// we are doing this inside the hook (we are always passing a specific locale)
await fetchGitHubContributors(relativePath, locale as Lang)
```

Reply:
Yep, can rename. Was partly me just keeping track of stuff, and the explicit label here helped clarify that this list being passed was already down to a specific locale, but I'm not picky; will rename to simplify.